src/util/sandbox/fd_sandbox.h

```diff
@@ -32,6 +32,13 @@
    only step that will be performed is switching UID and GID to the
    provided ones.  All other arguments are ignored.
 
+   The rlimit_nofile argument determines what RLIMIT_NOFILE is set to.
+   This should almost always be zero.  It does not affect already open
+   descriptors, and only is in effect for future syscalls.  It is
+   only needed for cases where poll() or similar syscalls are used which
+   require RLIMIT_NOFILE values to be increased for no particular
+   reason.
```

Comment:
It's possible `epoll` doesn't have this issue. I don't see anything about `RLIMIT_NOFILE` on the manpage for it

Reply:
Made a mental note to look into this, but leaving for a future change